tomcat-main/src/main/resources/defaults/base-procedure-transport.xml

```diff
@@ -0,0 +1,83 @@
+<record id="transport" in-findedit="yes" type="record,procedure" cms-type="default" generate-services-schema="true" >
+    <services-url>transports</services-url>
+    <services-tenant-plural>Transports</services-tenant-plural>
+    <services-tenant-singular>Transport</services-tenant-singular>
+    <services-list-path>abstract-common-list/list-item</services-list-path>
+    <services-record-path>transports_common:http://collectionspace.org/services/transport,transports_common</services-record-path>
+    <services-record-path id="collectionspace_core">collectionspace_core:http://collectionspace.org/collectionspace_core/,collectionspace_core</services-record-path>
+
+    <include src="domain-procedure-transport.xml" strip-root="yes" />
+
+    <section id="coreInformation">
+        <include src="core-fields.xml" strip-root="yes" />
+    </section>
+
+    <section id="transportInformation">
+        <field id="transportReferenceNumber" mini="number,list" />
+
+        <field id="transporter" autocomplete="true" />
```

Comment:
I would add `mini="summary,list"` here. When you do a search that returns multiple record types (e.g. search for All records, search for Procedures), the fields that have `mini=number` and `mini=summary` will be used as the "Record" and "Summary" columns in the search results. If you don't have both specified, one of the columns will be blank, which isn't an error per se, but since the column will be there regardless, you might as well put some potentially useful information there. If not otherwise specified, use a field that appears in the record title as the summary, which in this case is `transporter`.

(`list` means that the field should be returned in search results. I don't know where the name `mini` came from.)